apps/explorer/src/components/pagination/__test__/Pagination.test.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { describe, it, expect, vi } from 'vitest';
+
+import Pagination from '../Pagination';
+
+describe('Pagination test', () => {
+    it('check pagination buttons exist and check disabled behavior for the nextBtn and prevBtn', () => {
+        const { getByTestId, container } = render(
+            <Pagination totalItems={105} itemsPerPage={10} currentPage={1} />
+        );
+        const nextBtn = container.querySelector(
+            'button[data-testid="nextBtn"]:not([style*="display:none"])'
```

Comment:
I actually think `getByRole` was correct here, but that we didn't have a usable accessible name defined. I'd probably add `aria-label` to the next and previous buttons so that you can query against those instead of the unicode arrows.